pkg/profile/profile.go

```diff
@@ -0,0 +1,85 @@
+package profile
```

Comment:
For reference, and quick scanning, this is the new `profile` package that contains the logic for handling profiles.